PROTOCOL.md

```diff
@@ -0,0 +1,62 @@
+# Protocol
+
+### `/ws-star/2.0.0`
+
+## Connecting & Crypto challenge
+
+```protobuf
+message IdentifyRequest {
+  required string nonce = 1;
+}
+
+message IdentifyResponse {
+  required string id = 1;
+  required string pubKey = 2;
+  required bytes signature = 3;
```

Comment:
Why not have all the same fields as the [identify protocol](https://github.com/libp2p/js-libp2p-identify/blob/master/src/message.js#L26)?

Reply:
This is not the same identify protocol. Additionally in the libp2p-identify protocol the public keys never get exchanged.